frontend/src/components/questions-answers/QuestionCard.vue

```diff
@@ -3,17 +3,17 @@
         <div class="card-content">
             <div class="meta-fields">
                 <p class="arrow">&#8679;</p>
-                <p class="likes">{{ question.likes }}</p>
+                <p class="likes">{{ likes }}</p>
                 <p class="arrow">&#8681;</p>
             </div>
             <div class="content">
-                <router-link tag="h2" :to="{ name: 'question', query: { qid: String(question.id) }}">
-                    {{ question.title }}
+                <router-link tag="h2" :to="{ name: 'question', query: { qid: String(id) }}">
```

Comment:
change back to param for router-link here. query change was just to allow new questions.

Reply:
yes i've fixed this now